### ISSUE TITLE
Orientation sensor

### ISSUE THREAD

**Issue by willybanana** [mirsadm/motioncam#85]
Orientation sensor is too sensitive, it would be convenient to have a forced setting.
+ fix overlaps

![Screenshot_20211224-084133_Motion Cam](https://user-images.githubusercontent.com/47151206/147322251-0305329b-6aab-472c-b3a7-c8bad38440c0.jpg)
![Screenshot_20211224-084144_Motion Cam](https://user-images.githubusercontent.com/47151206/147322255-13f7a85d-a50a-4ce1-9e31-2c7cab445352.jpg)

**Comment by newillusion** [comment]
I have the issue with horisontal orientation. Samsung Note8.
P.S. Previous buttons (from first beta version) was better solution. 
![Screenshot_20211228-154807_Motion Cam](https://user-images.githubusercontent.com/13040758/147547570-e9a74fae-61ed-4f71-9923-980af2eb653a.jpg)

**Comment by mirsadm** [comment]
Yes that does need to be fixed. I am intending on putting buttons back but making them configurable. As I add more features they're not going to fit there but ultimately I imagine only a few things are useful without having to access the extended menu.

**Comment by 2blackbar** [comment]
I think now its cleaner, You can hide it with tap on that arrrow on left.
It would be cool to have a global setting to hide all these menus with shutter and iso etc when recording tho.
Not by default of course.Just a switch.

**Comment by newillusion** [comment]
> I am intending on putting buttons back but making them configurable.
Sounds great! It helps to time and attention economy (time is very valueable resource during video shooting). Thanks!
And if user can choose what exactly place on "hot" place - that is great twice...

**Comment by teemodk** [comment]
Not a big issue for photos, but it should not move during recording of videos. Especially for Zenfone 6/7/8; when lens is turned 90° (useful when recording Antz), the display is horizontal and it flickers very fast back and forth between orientation modes.

**Comment by mirsadm** [comment]
Yep I will to add an orientation lock, make is less sensitive and rotate to UI after I release the current build

**Comment by mirsadm** [comment]
Fixed the UI issues with the settings not being oriented correctly:
https://github.com/mirsadm/motioncam/releases/download/8.0.7-beta0/MotionCam.8.0.7-beta0.apk

I will make the orientation changes less sensitive/prevent oscillation in the next release
